xarray/core/dataset.py

```diff
@@ -996,7 +996,7 @@ def sel(self, method=None, tolerance=None, **indexers):
         )
         return self.isel(**pos_indexers)._replace_indexes(new_indexes)
 
-    def isel_points(self, dim='points', **indexers):
+    def isel_points_old(self, dim='points', **indexers):
```

Comment:
Can you remove the old function instead? That will make GitHub show a more informative diff.